code/game/objects/effects/decals/crayon.dm

```diff
@@ -551,24 +624,32 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 	sleep(30)
 	explosion(loc, 3, 5, 7, 5)
 	M.sanity.changeLevel(100)
+	// We log this for admins as it can be easily used for griefing and it causes quite the explosion.
+	log_and_message_admins("[M] has used the Paradox crayon spell, causing an explosion at \the [jumplink(M)] X:[M.x] Y:[M.y] Z:[M.z].")
 	return
 
+// The End: Removes your nearsighted disability, but also your knowledge of cult and occult languages
+// Basically wiping yourself clean of everything caused by crayon magic, but making you unable to cast spells again
 /obj/effect/decal/cleanable/crayon/proc/end_spell(mob/living/carbon/human/M)
 	var/datum/reagent/organic/blood/B = M.get_blood()
 	to_chat(M, "<span class='warning'>The truth of the universe flashes before your eyes at a sickening speed, eldritch knowledge being forcefully vacuumed out of your psyche. The light! It burns! IT BURNS!!!</span>")
-	M.maxHealth += 5
-	M.health += 5
-	M.disabilities &= ~NEARSIGHTED
+	M.disabilities &= ~NEARSIGHTED | ~BLIND
 	B.remove_self(150)
 	M.sanity.breakdown(TRUE)
 	M.sanity.changeLevel(5)
 	for(var/datum/language/L in M.languages)
 		if(L.name == LANGUAGE_CULT)
 			M.remove_language(LANGUAGE_CULT)
+			M.maxHealth += 20 // Give us back our health too
```

Comment:
thats not how this is meant to work, your meant to get 5 back, thats it.

Reply:
Sounds like a net loss for nothing, very ugly balance but fine.